content/en/docs/reference/glossary/volume.md

```diff
@@ -11,10 +11,10 @@ tags:
 - core-object
 - fundamental
 ---
- A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip term_id="pod" >}}.
+ A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.
 
 <!--more-->
 
-A Kubernetes volume lives as long as the Pod that encloses it. Consequently, a volume outlives any containers that run within the Pod, and data in the volume is preserved across container restarts.
+Kubernetes supports many types of volumes. Ephemeral volumes have a lifetime of a pod whereas Persistent volumes exist beyond the lifetime of a pod though for any kind of volume in a given pod, data is always preserved across container restarts.
```

Comment:
WDYT?
```suggestion
Kubernetes supports two fundamental types of volumes - ephemeral and persistent. 
Ephemeral volumes exist for the lifetime of their associated pods, while persistent 
volumes are designed to outlive the pods reading from them. However, irrespective
of the volume type, data is always preserved across container restarts.
```

Reply:
I think the changes makes it more clear and defined though i was wondering that at line 20, 
**persistent volumes are designed to outlive the  pods `reading` from  them.**
Please correct me if i'm wrong , Volumes are designed such that pods can perform the `CRUD` on it.
So to avoid the hassle we can simply go with `Persistent volumes are designed to exist beyond the lifetime of pods associated to it.`

Reply:
Good catch, thank you! Persistent volumes are designed to exist beyond the lifetime of pods associated with it sounds good, yes.

Reply:
Okay I'll make the commit with suggested changes.
Thank you for feedback and suggested changes @divya-mohan0209 .